### PR TITLE
Added volume action fix

### LINKED ISSUE #217: The volume action throwing error on some devices
**Describe the bug**
The volume action is not working on some devices and working on some devices

**To Reproduce**
1. `python3 -m main`
2. Say 'set audio to 80'

It will throw an error

**Expected behavior**
It should work on all devices

**Would you like to work on this issue**
Yes

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #217 

Changes 
* The volume action change was throwing an error on different versions of raspbian since we were directly using shell commands
* Now have implemented it through a python library

### CODE REVIEW

**Comment by stealthanthrax** [comment]
@prateekiiest , please have a look too

File: main/states/busy_state.py [hunk 1]
```diff
@@ -50,7 +50,10 @@ def on_enter(self, payload=None):
                     os.system('play ' + audio_url[6:])  # nosec #pylint-disable type: ignore
 
             if 'volume' in reply.keys():
-                subprocess.call(["amixer", "-D", "pulse", "sset", "Master", str(reply['volume'])])  # nosec #pylint-disable type: ignore
+                m = alsaaudio.Mixer()
+                m.setvolume(int(reply['volume']))
+                os.system('play {0} &'.format(
+                self.components.config['detection_bell_sound']))
```
Comment (prateekiiest) [inline main/states/busy_state.py]: Travis fails at this line due to missing indentation

File: main/states/busy_state.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 import os
 import pafy
 import subprocess   # nosec #pylint-disable type: ignore
-
+import alsaaudio
 
 class BusyState(State):
```
Comment (prateekiiest) [inline main/states/busy_state.py]: Travis fails here due to required 2 blanks

File: main/states/busy_state.py (hunk 1 above)
Comment (prateekiiest) [inline main/states/busy_state.py]: `detection_bell_sound` - does this play the bell sound when the volume has reached the desired level?

Comment (stealthanthrax) [reply main/states/busy_state.py]: Yes

**Comment by stealthanthrax** [comment]
@prateekiiest , fixed them

**Pull request merged**